source/user-manual/reference/ossec-conf/index_agent.rst

```diff
@@ -0,0 +1,105 @@
+.. Copyright (C) 2020 Wazuh, Inc.
+
+.. _reference_client_conf:
+
+Agent configuration (agent.conf)
+================================
+
+.. warning::
+    Upgrade from older version than v5.0.0, will create a new agent.conf file and current configuration will be saved into ossec.conf.backup file, restarting configuration from zero.
+
+The ``agent.conf`` file is the main configuration file on the Wazuh agent. It is located at ``/var/ossec/etc/agent.conf`` on Linux machines. On Windows agents, we can find it at ``C:\Program Files (x86)\ossec-agent\agent.conf``. It is recommended to back up this file before making changes on it. A configuration error may prevent Wazuh services from starting up.
+
+The ``agent.conf`` file is in XML format and all of its configuration options are nested in their appropriate section of the file. In this file, the outermost XML tag is ``<wazuh_config>``. There can be more than one ``<wazuh_config>`` tag.
+
+Here is an example of the proper location of the *active-response* configuration section:
+
+.. code-block:: xml
+
+    <wazuh_config>
+        <active-response>
+            <!--
+            active-response options here
+            -->
+        </active-response>
+    </wazuh_config>
+
+The ``shared/agent.conf`` file is very similar to ``agent.conf``, but ``shared/agent.conf`` is used to centrally distribute configuration information to agents. See more :doc:`here <../centralized-configuration>`.
+
+Wazuh can be installed in two ways: as a manager by using the "server/manager" installation type and as an agent by using the "agent" installation type.
+
++---------------------------------------------------------------------+------------------------+
+| Configuration sections                                              | Supported installations|
```

Comment:
I think the second column is not necessary.

Reply:
Done